### ISSUE TITLE
A11y Testing for vscodeedu.com - Text Spacing doesn't adjust for VS Code UI Toolkit buttons

### ISSUE THREAD

**Issue by IanMatthewHuff** [microsoft/vscode-webview-ui-toolkit#506]
Context: Pasted below is a the contents of a linked bug from vscode for edu testing. We use vscode.dev + the vscode webview UI toolkit to provide our UI. In a recent accessibility pass we had a couple of bugs filed that I believe the UI toolkit would need to either resolve or make the choice to not resolve for now.

For this issue the Google chrome extension "stylus" is used to bump up text spacing on the page and vscode UI tool kit does not respect this. FWIW I believe that this type of accessibility change would be handled by vscodes own line spacing settings

Original bug: https://github.com/microsoft/vscode-edu-extension/issues/1031

Text of Original bug:
### Describe the Bug

Text spacing is not adapting for "Back and Give feedback and Next Buttons" 
**Note:** Similar issue is observed in throughout application .

**Github Tags:**
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code For Education ; #GH-VisualStudioCodeforEducation-Web-Aug2023; #Textspacing; #MAS1.4.12; #A11yWCAG2.1; #Get Healthy; #ChromiumEdge; #Win11;

### Steps to Reproduce

1. Launch the application using URL: https://vscodeedu.com/. 
2. "Visual Studio Code for Education" screen will get open. 
3. TAB till "login" control and hit ENTER. 
4. Courses tab will get open. 
5. TAB till "My Courses" tab. 
6. TAB till "Start" control and hit ENTER. 
7. TAB till "Introduction to Python" control present in the left navigation and hit ENTER. 
8. TAB till "Unit 1" control and hit ENTER. 
9. TAB till "Start" control present in "Unit 1 - Getting Started" section and hit ENTER. 
10. "Welcome" screen will appear. 
11. TAB till "Next" control and hit ENTER until "Example: Mapping Question" screen appears. 
12. "Example: Mapping Question" screen will appear 
13. Verify whether that text spacing is not adapting for "Back and Give feedback and Next Buttons" or not 

### Expected behavior

Text spacing should be adapted for "Back and Give feedback and Next Buttons" also.

### Screenshots / Video

<img width="960" alt="A11y_Vs code for education_ Getting Started _Mapping Section _Text spacing " src="https://github.com/microsoft/vscode-edu-extension/assets/98530069/50e392bd-08aa-4d9c-a41e-ba0240b3ce01">
